### ISSUE TITLE
[a11y] cdk-focus-initial should be camelCased and lacks documentation

### ISSUE THREAD

**Issue by willshowell** [angular/material2#8006]
#### Bug, feature request, or proposal:
Bug

#### Expected

Via https://github.com/angular/material2/issues/6720, `cdk-focus-initial` should be `cdkFocusInitial`, since it is not a component.

#### Is there anything else we should know?

Maybe it could be a directive so that dgeni picks it up?

**Comment by willshowell** [comment]
Ah. Missed https://github.com/angular/material2/issues/6799.
